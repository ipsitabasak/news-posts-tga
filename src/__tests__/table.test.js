import React from 'react';
import Table from '../components/table';
import renderer from 'react-test-renderer';

describe('table component', () => {
  it('works correctly', () => {
    const component = renderer.create(
      <Table hits={[]} />,
    );
    let tree = component.toJSON();
  // again change
    expect(tree).toMatchSnapshot();
  });
}); 
