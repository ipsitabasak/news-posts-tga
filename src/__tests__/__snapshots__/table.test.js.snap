// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`table component works correctly 1`] = `
<div
  className="container"
>
  <ul>
    <li
      className="header"
    >
      <a
        className="title"
        href="/"
        onClick={[Function]}
        onMouseEnter={[Function]}
        style={
          Object {
            "borderColor": "black",
            "color": "black",
            "textDecoration": "none",
          }
        }
      >
        Y
      </a>
      <a
        className="title-spacing"
        href="/top"
        onClick={[Function]}
        onMouseEnter={[Function]}
        style={
          Object {
            "borderColor": "black",
            "color": "black",
            "textDecoration": "none",
          }
        }
      >
        top
      </a>
      <span>
        |
      </span>
      <a
        className="title-spacing"
        href="/new"
        onClick={[Function]}
        onMouseEnter={[Function]}
        style={
          Object {
            "borderColor": "black",
            "color": "black",
            "textDecoration": "none",
          }
        }
      >
        new
      </a>
    </li>
    <li
      className="footer"
    >
      <button
        className="more-button"
        onClick={undefined}
      >
        MORE
      </button>
    </li>
  </ul>
  <style
    jsx={true}
  >
    
            .title {
              padding: 5px;
              display: inline-block;
              border: 1px solid white;
              margin: 3px 5px 0 5px;
              font-size: 10px;
            }
            .title-spacing {
              padding: 0 10px;
            }
            .container {
              display: flex;
              justify-content: center;
              font-family: Arial;
              letter-spacing: 0.2px;
            }
            ul {
              background-color: #F6F6EF;
              max-width: 1440px;
              padding: 0;
            }
            .header {
              display: inline-block;
              width: 100%;
              text-align: left;
              height: 30px;
              margin-bottom: 10px;
              background-color: #ff6600;
              font-size: 12px;
            }
            .footer {
              display: inline-block;
              width: 100%;
              text-align: left;
              height: 30px;
            }
            .more-button {
              background: none;
              border: none;
              margin-left: 120px;
            }
          
  </style>
</div>
`;
